ui/DesignSystem/Component/SourceBrowser/Readme.svelte

```diff
@@ -0,0 +1,64 @@
+<script>
+  import { Icon } from "../../Primitive";
+
+  export let blob = null;
+  export let path = null;
+
+  console.assert(blob.binary === false, "blob should not be binary");
```

Comment:
We don't have any telemetry set up, so we won't know this happened in the wild.
If somebody actually saves a binary `README.md` file, should the UI just not show the "binary" placeholder?

Reply:
That's already the case, since the Readme component won't be called if the file is binary. This is here as documentation, and to catch coder errors.

Reply:
See https://github.com/radicle-dev/radicle-upstream/pull/311/files#diff-b8ce47c84b0bdc9f5a5c2240deef503bR214-R222

Reply:
https://github.com/radicle-dev/radicle-upstream/blob/d442e5d2c6f54e525d5e5f3c5d20bb07a4a14342/ui/Screen/Project/Source.svelte#L214-L222